src/incidents/list/ViewIncidentsTable.tsx

```diff
@@ -22,33 +23,98 @@ function ViewIncidentsTable(props: Props) {
     return <Spinner type="DotLoader" loading />
   }
 
+  // filter data
+  const exportData = [{}]
+  let first = true
+  if (data != null) {
+    data.forEach((elm) => {
+      const entry = {
+        code: elm.code,
+        date: format(new Date(elm.date), 'yyyy-MM-dd hh:mm a'),
+        reportedBy: elm.reportedBy,
+        reportedOn: format(new Date(elm.reportedOn), 'yyyy-MM-dd hh:mm a'),
+        status: elm.status,
+      }
+      if (first) {
+        exportData[0] = entry
+        first = false
+      } else {
+        exportData.push(entry)
+      }
+    })
+  }
+
+  function downloadCSV() {
+    const fields = Object.keys(exportData[0])
+    const opts = { fields }
+    const parser = new Parser(opts)
+    const csv = parser.parse(exportData)
+    console.log(csv)
+
+    const filename = 'IncidenntsCSV.csv'
```

Comment:
May you correct the spelling if "Incidents".
Perhaps this can be defined in the locale file?

Reply:
on it.

Reply:
done

Reply:
Awesome! thanks @reidmeyer